Web/starter_project_barbies/data/about.ts

```diff
@@ -0,0 +1,75 @@
+import { ProblemItem, Project, Session } from '@/types/AboutTypes'
+
+export const Projects: Project[] = [
+  {
+    image: '/about/hakimhub.png',
```

Comment:
none of the links should have the `/about` in them
you should decide that when you are rendering the page because it is common for all)
it also gives the flexibility to change that folder without having to modify all these links